nextstrain/cli/command/login.py

```diff
@@ -0,0 +1,153 @@
+"""
+Logs in a Nextstrain.org groups user, storing their credentials locally.
```

Comment:
Minor thing, but I've been capitalizing Nextstrain Groups (and even just when "Groups" in passing) as a way to indicate that it's a capital-T Thing and distinguish it from the more general meaning of "group".